src/modules/ui/directives/change24/Change24.js

```diff
@@ -31,17 +31,24 @@
                  * @type {number}
                  */
                 this.interval = null;
+                /**
+                 * @type {boolean}
+                 */
+                this.directionByPair = null;
             }
 
             $postLink() {
                 this.precision = Number(this.precision) || 2;
                 this.interval = Number(this.interval) || 5000;
 
                 if ($attrs.noUpdate) {
-                    this._getChange().then(this._setChange.bind(this), this._setChange.bind(this));
-                    this.observe(['assetFrom', 'assetTo'], () => {
+                    const change = this._getChange();
```

Comment:
This variable stores value only to pass it to the if-statement on the next line. It might be inlined without significant loss.